src/test/java/org/folio/circulation/support/http/server/ValidationErrorTest.java

```diff
@@ -0,0 +1,81 @@
+package org.folio.circulation.support.http.server;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasKey;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.nullValue;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+class ValidationErrorTest {
+
+  @Test
+  void oneParam() {
+    var e = new ValidationError("my.code", "foo {a} bar", "a", "b");
+    assertThat(e.getCode(), is("my.code"));
+    assertThat(e.getMessage(), is("foo b bar"));
+    assertThat(e.getParameter("a"), is("b"));
+    assertThat(e.toJson(), is(
+        new JsonObject()
+        .put("code", "my.code")
+        .put("message", "foo b bar")
+        .put("parameters", new JsonArray().add(
+            new JsonObject().put("key", "a").put("value", "b")))));
+  }
+
+  @Test
+  void oneParamWithNullMessage() {
+    var e = new ValidationError("my.code", null, "a", "b");
```

Comment:
I believe this would produce a error that is invalid based upon the JSON [schema](https://github.com/folio-org/raml/blob/3126cb6ab63ec9237ccb88c436fadb1734e9e226/schemas/error.schema#L25) used for validation errors.

Given we are demonstrating this behaviour explicitly in this test, I think we should either
* change the test (and implementation) to disallow validation errors without a message
* change the JSON schema to allow errors without a message

Reply:
Good catch, thanks! Fixed.